openrr-nav-viewer/src/parser.rs

```diff
@@ -57,3 +58,24 @@ pub fn parse_robot_pose_to_point(robot_pose: &Pose, color: Color32, point_radius
 
     Points::new(plot_point).color(color).radius(point_radius)
 }
+
+pub fn parse_robot_pose_to_polygon(robot_pose: &Pose, color: Color32, size: f64) -> Polygon {
```

Comment:
I don't think `parse` is the best name for this function.
I think `robot_pose_to_polygon` is enough. (Or use `convert_robot_pose_to_polygon`?)